### ISSUE TITLE
Xcode 13 beta 5 fails to build GRDB in Release configuration with CocoaPods

### ISSUE THREAD

**Issue by foxware00** [groue/GRDB.swift#1033]
### What did you do?

Compile a release build on XCode Version 13.0 beta 5 (13A5212g)

Fully aware it's a beta and likely to break, but just incase it's a long term change. It seems to be different from #984, but feel free is close if it's a duplicate.

### What happened?

Compliation failed, please see attached screenshot of logs. Happy to provide anything else that may be useful. It seems the 
`#if canImport(Combine)` at the top of DatabaseMigrator.swift seems to be the cuprit.

![Screenshot 2021-08-18 at 11 36 51](https://user-images.githubusercontent.com/4174528/129884697-6f8f528b-f098-45f0-babb-c06b3ecc973a.png)

### Environment

**GRDB flavor(s):** (GRDB)
**GRDB version:** 5.9.0
**Installation method:** CocoaPods
**Xcode version:** Version 13.0 beta 5 (13A5212g)
**Swift version:** 1.26.9 Apple Swift version 5.5 (swiftlang-1300.0.29.102 clang-1300.0.28.1)
**Platform(s) running GRDB:** iOS
**macOS version running Xcode:**: 11.5.2 (20G95)

**Comment by groue** [comment]
Hello @foxware00,

I can reproduce your error, when I attempt at archiving an app. Development builds, however, work fine.

It looks a lot like https://github.com/realm/realm-cocoa/issues/7368

**Comment by foxware00** [comment]
> Hello @foxware00,
> 
> I can reproduce your error, when I attempt at archiving an app. Development builds, however, work fine.
> 
> It looks a lot like [realm/realm-cocoa#7368](https://github.com/realm/realm-cocoa/issues/7368)

Not a long term solution, but commenting out any `#if canImport(Combine)` and related code allows compliation as I'm not using Combine anyway.

**Comment by groue** [comment]
> Not a long term solution, but commenting out any `#if canImport(Combine)` and related code allows compliation as I'm not using Combine anyway.

I'm glad it's a solution for you during the beta phase. GRDB needs conditional import of Combine, though, due to the range of operating systems it targets.

**Comment by groue** [comment]
According to https://github.com/realm/realm-cocoa/issues/7368#issuecomment-895296330 the issue has already been reported to Apple.

**Comment by foxware00** [comment]
@groue Thank you Groue, thought id' leave the workaround for others, I had a hunch is was on apple to fix. Many thanks.

**Comment by groue** [comment]
See also https://github.com/Alamofire/Alamofire/issues/3481

**Comment by sobri909** [comment]
The workaround from the [Apple Forums thread](https://developer.apple.com/forums/thread/682285) works too. That is, changing the iOS platform version to 13.0 in the Package.swift file. 

```swift
let package = Package(
    name: "GRDB",
    platforms: [
        .iOS("13.0"),
        ...
```

Obviously not a proper solution, but it gets a path to a working build with only one line changed, until Apple figure it out at their end.

**Comment by Patrik-svobodik** [comment]
@groue would you mind making a iOS 13+ only branch for now? This issue has still not been fixed and my team needs to do some profiling as we're coming close to the RC of iOS 15

**Comment by groue** [comment]
Hello @Patrik-svobodik, thanks for asking. I'm away from any computer for one more week, unable to perform such task.

Would you please help, and write a short HOWTO that could guide clueless people? Don't be afraid staying the obvious, and just make sure the recipe can be applied, step by step, by anyone. This will help many people.

**Comment by sobri909** [comment]
My steps: 

1. Fork the repo and clone the fork locally
2. In the local copy of the fork, modify the `Package.swift`, changing line 9 to `.iOS("13.0"),` 
3. Drag the fork's folder into the Xcode project files list, to use it instead of the main repo

Uh, I just realised I didn't need to use a fork for a team of one. But yeah, for a team to use you'll want a fork, so that you can commit the change and use the fork as the project dependency instead of the main repo.

**Comment by Patrik-svobodik** [comment]
I’m sorry for forgetting, I missed the email @groue, should I still make the document later today?

Another way I’ve found to work around the problem is to use CocoaPods

**Comment by groue** [comment]
No @Patrik-svobodik, @sobri909 wrote a recipe right above your message. And the Cocoapods workaround looks like a recipe on its own.

Thanks, and happy GRDB!

**Comment by guidedways** [comment]
The recipe works but 5.11.0 as it is fails to compile under Xcode 13.0 RC.

**Comment by MoSchaub** [comment]
maybe consider the solution used by realm [realm/realm-cocoa#7401](https://github.com/realm/realm-cocoa/pull/7401)

**Comment by groue** [comment]
The issue should be fixed in GRDB v5.12.0, which has dropped support for 32-bits devices by bumping the minimum iOS version to 11.0.
